scanner/src/main/kotlin/experimental/NestedProvenanceResolver.kt

```diff
@@ -42,7 +43,10 @@ interface NestedProvenanceResolver {
 /**
  * The default implementation of [NestedProvenanceResolver].
  */
-class DefaultNestedProvenanceResolver(private val workingTreeCache: WorkingTreeCache) : NestedProvenanceResolver {
+class DefaultNestedProvenanceResolver(
```

Comment:
Unfortunately not, due to git-repo using the path for the manifest file.

Reply:
What do you think about doing a case distinction? (To me this seems more consistent with the scan by repo approach, where we move away from sparse checkouts which in the end is exactly that, ignoring the VCS path for non-Git-Repo projects to maximize the cache hits.

Reply:
I would like to address the whole VCS path topic in a separate PR, ideally after fixing the git-repo manifest path issue so that `vcs.path` is not ambiguous anymore.